src/components/pages/main-page/MainPage.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from 'react';
+import { Pagination } from '@mui/material';
+
+import CourseCard from '@/components/common/ui/CourseCard/CourseCard';
+import { useAppDispatch, useAppSelector } from '@/hooks/useSelect';
+import { fetchCourses } from '@/redux/courses/asyncActions';
+import { selectDetails } from '@/redux/courses/selectors';
+
+import styles from './MainPage.module.scss';
+
+const MainPage = () => {
+  const [pageNumber, setPageNumber] = useState(0);
+  const coursesPerPage = 10;
+  const pagesVisited = pageNumber * coursesPerPage;
```

Comment:
I think the page number should start with 1. It makes the code a little clearer and understandable. 
P.S. Remove also ```-1``` in  setPageNumber(value -1) and Pagination ```page``` prop below